package.json

```diff
@@ -14,17 +14,17 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "16.2.12",
-    "@angular/cdk": "16.2.11",
-    "@angular/common": "16.2.12",
-    "@angular/compiler": "16.2.12",
-    "@angular/core": "16.2.12",
-    "@angular/fire": "^7.6.1",
-    "@angular/forms": "16.2.12",
-    "@angular/platform-browser": "16.2.12",
-    "@angular/platform-browser-dynamic": "16.2.12",
-    "@angular/router": "16.2.12",
-    "@fortawesome/angular-fontawesome": "^0.13.0",
+    "@angular/animations": "17.0.6",
+    "@angular/cdk": "17.0.3",
+    "@angular/common": "17.0.6",
+    "@angular/compiler": "17.0.6",
+    "@angular/core": "17.0.6",
+    "@angular/fire": "^17.0.0",
```

Comment:
Additional prework for this PR: update this package and its dependencies (tools and admin packages) to work for v16